### ISSUE TITLE
Loki-Docker-Driver doesn't work with Docker 1.13.1

### ISSUE THREAD

**Issue by ArthurSens** [grafana/loki#866]
**Describe the bug**
Loki-Docker-Driver doesn't work with Docker 1.13.1

**To Reproduce** 
Install docker
Install loki-docker-driver plugin
Try to run any container using loki as log-driver
```shell
yum install -y yum-utils
yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
yum install -y docker
systemctl enable docker 
systemctl start docker
ln -s /usr/libexec/docker/docker-runc-current /usr/libexec/docker/docker-runc
docker plugin install  grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
docker run --log-driver=loki \
    --log-opt loki-url="http://my.loki.server/api/prom/push" \
    --log-opt loki-retries=5 \
    --log-opt loki-batch-size=400 \
    grafana/grafana
```
I'll get the following error
```
Unable to find image 'grafana/grafana:latest' locally
Trying to pull repository docker.io/grafana/grafana ... 
latest: Pulling from docker.io/grafana/grafana
7413c47ba209: Pull complete 
0fe7e7cbb2e8: Pull complete 
1d425c982345: Pull complete 
344da5c95cec: Pull complete 
733f63dd777b: Pull complete 
932f3f18744a: Pull complete 
3068efe67dfa: Pull complete 
f23ece496408: Pull complete 
9e2a3cc28abf: Pull complete 
Digest: sha256:11a6bd1bd67853acfc14ec6187cd26e723b1a97dba52e50dc5fc2ba6428f7028
Status: Downloaded newer image for docker.io/grafana/grafana:latest
/usr/bin/docker-current: Error response from daemon: logger: no log driver named 'loki' is registered.
See '/usr/bin/docker-current run --help'.
```

**Expected behavior**
Container starts as it should be and send logs to loki

**Environment:**
```shell
[root@myserver ~]# cat /etc/os-release 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

[root@myserver ~]# docker version
Client:
 Version:         1.13.1
 API version:     1.26
 Package version: docker-1.13.1-102.git7f2769b.el7.centos.x86_64
 Go version:      go1.10.3
 Git commit:      7f2769b/1.13.1
 Built:           Mon Aug  5 15:09:42 2019
 OS/Arch:         linux/amd64

Server:
 Version:         1.13.1
 API version:     1.26 (minimum version 1.12)
 Package version: docker-1.13.1-102.git7f2769b.el7.centos.x86_64
 Go version:      go1.10.3
 Git commit:      7f2769b/1.13.1
 Built:           Mon Aug  5 15:09:42 2019
 OS/Arch:         linux/amd64
 Experimental:    false

```

**Aditional Information**

I've tried the same thing but changing the docker version to `docker-ce-18.06.1.ce-3.el7` and everything works as expected

**Comment by adityacs** [comment]
@ArthurSens Issue might be due to docker [error](https://github.com/moby/moby/issues/35553). It's fixed only in 18.0.3-ce. Kindly refer [here](https://github.com/docker/docker-ce/releases?after=v18.06.1-ce-rc1)

**Comment by ArthurSens** [comment]
> @ArthurSens Issue might be due to docker [error](https://github.com/moby/moby/issues/35553). It's fixed and released only in 18.0.3-ce. Kindly refer [here](https://github.com/docker/docker-ce/releases?after=v18.06.1-ce-rc1)

The error mentioned in this issue is a little different, because he was able to run a docker container with `docker run` at least once. But yeah, I think it is a docker issue.

Sadly, I can't upgrade the docker version to community edition, so I think I'll wait for the promtail image with journal support, mentioned [here](https://github.com/grafana/loki/issues/872#issuecomment-519498763).

Thank you for the helpful answer and for this awesome tool

**Comment by rfratto** [comment]
@ArthurSens the promtail image with journal support was released in v0.3.0, are you still having problems getting Loki up and running?

**Comment by ArthurSens** [comment]
I'm currently on vacations, when I get back to work(2 weeks from now) I'll test it. 

But feel free to close the issue, the problem I reported is not Loki related
